gdcdictionary/schemas/sample.yaml

```diff
@@ -728,6 +422,26 @@ properties:
     enum:
       - host-associated
 
+# Prop originated from ncbi-covid-19 project
+  country_region:
+    description: "Name of the country or region where the sample was collected."
+    type: string
+
+# Prop originated from ncbi-covid-19 project
+  city_province:
```

Comment:
This prop should be "province_state"

Reply:
property name and description was changed